metadata_layer.py

```diff
@@ -352,93 +352,492 @@ def __repr__(self):
         )
 
 class FormatData(object):
-    def __init__(self, content_type, drm_scheme, link=None):
+    def __init__(self, content_type, drm_scheme, link=None, rights_uri=None):
         self.content_type = content_type
         self.drm_scheme = drm_scheme
         if link and not isinstance(link, LinkData):
             raise TypeError(
                 "Expected LinkData object, got %s" % type(link)
             )
         self.link = link
+        self.rights_uri = rights_uri
```

Comment:
It's currently not used at all. I was planning for what has become the next branch after this one, in which RightsStatus adheres to the DeliveryMechanism, not to the LicensePool. (or not _just_ to the LicensePool)